src/mscorlib/src/System/Span.cs

```diff
@@ -15,18 +15,13 @@ namespace System
     /// </summary>
     public struct Span<T>
     {
-        /// <summary>A byref or a native ptr.</summary>
```

Comment:
The convention is to keep fields at the top. (It was recently added to https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md to make it explicit.)